src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/order_actions.html.twig

```diff
@@ -62,11 +64,79 @@
   {% if orderForViewing.documents.canGenerateDeliverySlip %}
     <form class="form-inline d-inline-block form-horizontal ml-2">
       <div class="input-group">
-        <a href="{{ path('admin_orders_generate_delivery_slip_pdf', {'orderId': orderForViewing.id}) }}" class="btn btn-action">
+        <a href="{{ path('admin_orders_generate_delivery_slip_pdf', {'orderId': orderForViewing.id}) }}"
+           class="btn btn-action">
           <i class="material-icons">local_shipping</i>
           {{ 'View delivery slip'|trans({}, 'Admin.Orderscustomers.Feature') }}
         </a>
       </div>
     </form>
   {% endif %}
+
+  {# Rendering of hook actionGetBackOfficeOrderButtons, see ActionsBarButtonsCollection #}
+  {% if (backOfficeOrderButtons.isEmpty() != true) %}
```

Comment:
If this kind of block is going to be on many pages, it should be interesting to create a macro to do it :)

Reply:
Only here so far 🤔